pkg/agent/controller/noderoute/node_route_controller.go

```diff
@@ -461,6 +461,10 @@ func (c *Controller) addNodeRoute(nodeName string, node *corev1.Node) error {
 		klog.Errorf("Failed to retrieve IP address of Node %s: %v", nodeName, err)
 		return nil
 	}
+	peerNodeMAC, err := GetNodeMAC(node)
```

Comment:
It needs to handle the MAC update, otherwise it will always use the empty annotation as the annotation is set asynchronously.

Reply:
Updated.